plugins/maciVoting/pages/proposal.tsx

```diff
@@ -12,16 +12,16 @@ import { useCoordinator } from "../hooks/useCoordinator";
 import { useCanFinalize } from "../hooks/useCanFinalize";
 
 export default function ProposalDetail({ id: proposalId }: { id: string }) {
-  const { proposal, status } = useProposal(proposalId, true);
-  const showProposalLoading = getShowProposalLoading(proposal, status);
+  const { proposal, proposalMetadata, creator, status } = useProposal(proposalId, true);
```

Comment:
if we add the creator address in the metadata when creating a new proposal (in the maci-voting-aragon-plugin repo) we could only use one value: proposalMetadata